dcc-download-server/src/test/java/org/icgc/dcc/download/server/fs/AbstractFileSystemViewTest.java

```diff
@@ -118,7 +119,7 @@ private static void assertCreationDate(File srcFile, long actualDate) {
     val fileAttributes = getFileAttributes(srcFile);
     val creationTime = fileAttributes.creationTime();
     log.info("{}", creationTime);
-    assertThat(actualDate).isEqualTo(creationTime.toMillis());
+    assertThat(isEpochsEqualUpToSecondsDigits(actualDate, creationTime.toMillis())).isTrue();
```

Comment:
Open to other suggestions on how to deal with this epoch time precision issue :thinking: